src/tests/suite/daos_container.c

```diff
@@ -52,7 +54,28 @@ co_create(void **state)
 				    &info, arg->async ? &ev : NULL);
 		assert_rc_equal(rc, 0);
 		WAIT_ON_ASYNC(arg, ev);
+		assert_int_equal(info.ci_nhandles, 0);
```

Comment:
not quite sure i get why this should be 0 and not 1. we just opened 1 handle.

Reply:
Like for the fetching of the container metadata (open, modify) times I decided that the number of open handles returned should reflect the state before the open, instead of including the effect of the open.

Reply:
this is counter intuitive IMO. idk about the time values, but for this particular count, for a user we really should return the actual count of open handles at that point where they are doing the checking. the user has just opened the container. getting back 0 open handle count sounds wrong IMO.

Reply:
Yes it is counter-intuitive. I'm going to look at returning nhandles to reflect the effect of the current open being executed.

And while doing this I will review the container metadata open time and whether it can be pre-updated to be consistent with what will be done with nhandles here. Open time has one tricky special case to account for (no update at all) - perhaps it is that case that made it seemingly difficult to pre-update the open time (when we do want the update to occur).

Reply:
I have local commits that both pre-increment nhandles, and  pre-update the open time - will have both in the next push